google-cloud-asset/samples/acceptance/quickstart_test.rb

```diff
@@ -58,6 +58,7 @@
 
   after do
     delete_bucket_helper bucket.name
+    delete_dataset_helper dataset.dataset_id
```

Comment:
Can you instead keep this scoped to only where the dataset is actually used? Otherwise an unnecessary dataset will be created and deleted for every test, putting more strain on resources.

Reply:
Done.